### PR TITLE
Add new sections

### PR DESCRIPTION
**Pull request by ashu8912**
This patch adds ```LimitRange```, ```Leases``` and ```ResourceClass``` sections which can be accessed from  ```Configuration``` menu in sidebar

Acceptance Criteria

- [x] We have a List and detail view for LimitRange
- [x] We have a List and detail view for Leases
- [x] We have a List and detail view for ResourceClass
- [x] We have test for all the above views

### CODE REVIEW

File: frontend/src/components/lease/Details.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+import { Lease } from '../../lib/k8s/lease';
+import { DateLabel, DetailsGrid } from '../common';
+
+export function LeaseDetails() {
+  const { namespace, name } = useParams<{ namespace: string; name: string }>();
+  const { t } = useTranslation('glossary');
```
Comment (joaquimrocha) [inline frontend/src/components/lease/Details.tsx:8]: This should be in the `resource` namespace, not glossary, I think. glossary is normally for the type of things and other technical jargon.

File: frontend/src/components/lease/List.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+import { Lease } from '../../lib/k8s/lease';
+import { SectionBox, SectionFilterHeader } from '../common';
+import ResourceTable from '../common/Resource/ResourceTable';
+
+export function LeaseList() {
+  const { t } = useTranslation(['glossary', 'leases']);
```
Comment (joaquimrocha) [inline frontend/src/components/lease/List.tsx:7]: Let's use `resource` instead of `leases`.

File: frontend/src/components/lease/List.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <SectionBox title={<SectionFilterHeader title={t('glossary|Lease')} />}>
+      <ResourceTable
+        resourceClass={Lease}
+        columns={[
+          'name',
+          'namespace',
+          {
+            label: t('frequent|Holder'),
```
Comment (joaquimrocha) [inline frontend/src/components/lease/List.tsx:16]: You don't have frequent in the pre-load of useTranslation.

File: frontend/src/components/runtimeClass/Details.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+import { RuntimeClass } from '../../lib/k8s/runtime';
+import { DetailsGrid } from '../common';
+
+export function RuntimeClassDetails() {
+  const { t } = useTranslation('glossary');
```
Comment (joaquimrocha) [inline frontend/src/components/runtimeClass/Details.tsx:7]: Use resource namespace.

File: frontend/src/components/limitRange/Details.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      namespace={namespace}
+      withEvents
+      extraInfo={item =>
+        item && [
+          {
+            name: t('Container Limits'),
+            value: (
+              <>
+                <Box m={1}>
+                  <Typography variant="h6">{t('Default')}</Typography>
```
Comment (joaquimrocha) [inline frontend/src/components/limitRange/Details.tsx:23]: This and the strings below should be in the `frequent` namespace I think.

File: frontend/src/components/limitRange/Details.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  return (
+    <DetailsGrid
+      resourceType={LimitRange}
+      name={name}
+      namespace={namespace}
+      withEvents
+      extraInfo={item =>
+        item && [
+          {
+            name: t('Container Limits'),
```
Comment (joaquimrocha) [inline frontend/src/components/limitRange/Details.tsx:19]: This should be in the resource namespace.

File: frontend/src/components/limitRange/Details.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
+import { Box, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+import { LimitRange } from '../../lib/k8s/limitRange';
+import { DetailsGrid, MetadataDictGrid } from '../common';
+
+export function LimitRangeDetails() {
+  const { t } = useTranslation('glossary');
```
Comment (joaquimrocha) [inline frontend/src/components/limitRange/Details.tsx:8]: Please use the `resource` and `frequent` namespaces.

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
Left a few comments about the translation namespaces but otherwise it looks good.

File: frontend/src/components/limitRange/Details.tsx (hunk 7 above)
Comment (ashu8912) [reply frontend/src/components/limitRange/Details.tsx]: fixed

File: frontend/src/components/limitRange/Details.tsx (hunk 6 above)
Comment (ashu8912) [reply frontend/src/components/limitRange/Details.tsx]: fixed

File: frontend/src/components/limitRange/Details.tsx (hunk 5 above)
Comment (ashu8912) [reply frontend/src/components/limitRange/Details.tsx]: fixed

File: frontend/src/components/runtimeClass/Details.tsx (hunk 4 above)
Comment (ashu8912) [reply frontend/src/components/runtimeClass/Details.tsx]: updated

File: frontend/src/components/lease/List.tsx (hunk 3 above)
Comment (ashu8912) [reply frontend/src/components/lease/List.tsx:16]: updated

File: frontend/src/components/lease/List.tsx (hunk 2 above)
Comment (ashu8912) [reply frontend/src/components/lease/List.tsx]: fixed

File: frontend/src/components/lease/Details.tsx (hunk 1 above)
Comment (ashu8912) [reply frontend/src/components/lease/Details.tsx]: updated

**Review by joaquimrocha — APPROVED** [review]
Just another namespace preload missing, but after that I think it's good!

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**